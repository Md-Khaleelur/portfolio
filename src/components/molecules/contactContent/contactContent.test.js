import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ContactContent from './contactContent';

describe("ContactContent component",() =>{
    test("Matches the snapshot",() => {
        const contactContent = create(<ContactContent/>);
        expect(contactContent.toJSON()).toMatchSnapshot();

    })
    test("ContactContent is rendered",() => {
        const { getByTestId, getByText } = render(<ContactContent/>);
        expect(getByTestId("contactContent")).toBeInTheDocument();
        
    });
   

});