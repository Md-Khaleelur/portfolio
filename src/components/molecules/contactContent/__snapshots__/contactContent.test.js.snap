// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactContent component Matches the snapshot 1`] = `
<div
  data-testid="contactContent"
>
  <div
    data-testid="title"
  >
    <h2>
      Contact US
    </h2>
  </div>
  <div
    data-testid="para"
  >
    <p>
      Contact US page body content
    </p>
  </div>
</div>
`;
